src/Widgets/FormStepContainerWidget/FormStepContainerWidgetComponent.tsx

```diff
@@ -129,6 +154,7 @@ Scrivito.provideComponent(FormStepContainerWidget, ({ widget }) => {
 
     indicateProgress();
     try {
+      //TODO: add token for Neoletter when Neoletter is ready to use it.
```

Comment:
`//TODO: add token for Neoletter when Neoletter is ready to use it.` i think it might be already ready

Reply:
Right, i removed the TODO